client/shipping-label/state/selectors/errors.js

```diff
@@ -58,7 +58,22 @@ const getPackagesErrors = ( values ) => _.mapValues( values, ( pckg ) => {
 	return errors;
 } );
 
-const getRatesErrors = ( values ) => _.mapValues( values, ( ( rate ) => rate ? null : __( 'Please choose a rate' ) ) );
+const getRatesErrors = ( { values: selectedRates, available: allRates } ) => {
+	return {
+		server: _.mapValues( allRates, ( rate ) => {
+			if ( rate.errors ) {
+				return rate.errors.map( ( error ) => {
+					return {
+						message: ( error.userMessage ||
+							error.message ||
+							"We couldn't get a rate for this package, please try again." ),
+					};
+				} );
+			}
+		} ),
```

Comment:
Challenge accepted 😉 One indentation level was removed when using a string instead of an error object, and I removed two more in 15a2063739c7f756a31f77aa3d6a13aefd3fa8d0